gobblin-core/src/test/java/org/apache/gobblin/converter/avro/JsonIntermediateToAvroConverterTest.java

```diff
@@ -121,4 +144,22 @@ public void testConverter()
 
     Assert.assertNotEquals(record.get("LastModifiedDate"), record2.get("LastModifiedDate"));
   }
+
+  @Test
+  public void testComplexSchema1()
```

Comment:
These tests fail for me as well.

Reply:
@jack-moseley tests are passing for me. Were the resources loaded when tests ran? Could you share the failing message?

Reply:
@jack-moseley found the failing test